test/e2e/testdata/template-failed-metric.yaml

```
@@ -0,0 +1,54 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: dag-task-test
```

```suggestion
  name: template-failed-metric
```

by convention, these have the same name as their filename

---

Although I'd recommend a more specific name for both the file and the `name`:

```suggestion
  name: template-status-failed-conditional-metric
```